_layouts/indicator.html

```diff
@@ -73,17 +75,17 @@ <h4>{{ t.indicator.sub_categories }}</h4>
         <div class="async-loaded" style="display:none">
           <ul class="nav nav-tabs data-view" role="tablist">
             <li role="presentation" class="nav-item active">
-              <a class="nav-link" data-toggle="tab" href="#chartview" aria-controls="chartview" role="tab">{{ t.indicator.chart }}</a>
+              <a class="nav-link" data-toggle="tab" href="#chartview" onClick='ga("send", "event", "Tab change", "Change data view", "Change to Chart tab")' aria-controls="chartview" role="tab" show-controls="1">{{ t.indicator.chart }}</a>
```

Comment:
We'll want to remove the ga() stuff, since this ticket is not related to Google Analytics.

And as above probably `data-show-controls` instead of `show-controls`.